app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -130,6 +134,17 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
     return target != null && target[kCiYamlTargetIsFlakyKey] == true;
   }
 
+  bool _getIgnoreFlakiness(String builderName, YamlMap ci) {
```

Comment:
I'd like to keep this PR as simple and in place as possible. And do the updates in a future PR.

Reply:
2 of the 3 files uses the YAML objects, which means, we would have to swap out that whole mechanism with the schedulerConfigs. That increases chances of bugs. It might be better to do an immediate follow up PR if we want to do the optimization.

Reply:
SGTM with an immediate follow up PR. Thanks!

Reply:
Thanks, will start with the new PR.